templates/partials/question.html

```diff
@@ -58,12 +58,16 @@
 {% set question_guidance %}
   {%- if question.guidance -%}
     {% set contents = question.guidance.contents %}
-    {% call onsPanel({
-      "id": "question-guidance-" ~ question.id,
-      "classes": "ons-u-mb-m"
-    }) %}
-      {% include 'partials/contents.html' %}
-    {% endcall %}
+    {% for item in contents %}
+      {%- if item["title"] or item["description"] or item["list"] -%}
```

Comment:
would `item.title`, `item.description` and `item.list` not work here?

Reply:
thats really interesting I didn't actually realise it was different - https://jinja.palletsprojects.com/en/3.0.x/templates/#variables

It should only be a difference in the order it tries to find the variable, I tried adding 
```
{{ item["title"]|pprint }}
<br/>
{{ item.title|pprint }}
```
above this and it printed the same my end, but in any case, square brackets look for the item before the attribute which isn't there so what you have is better anyway 👀 , my bad